src/config/app/app-config.js

```diff
@@ -44,6 +44,29 @@ const config = {
     clearInvalid: false,
     strictHeader: false,
     isSameSite: 'Strict'
+  },
+  local: process.env.LOCAL || false,
+  serviceBus: {
+    environment: process.env.NODE_ENV || 'development',
+    msgSrc: 'ffc-grants-eligibility-checker',
+    type: 'queue',
+    useCredentialChain: true,
+    host: process.env.SERVICE_BUS_HOST || 'live-service-bus-host',
+    notifyEmailTemplate: process.env.NOTIFY_EMAIL_TEMPLATE || 'live-local-notify-template',
+    queueId: process.env.NOTIFY_SUBMIT_QUEUE || 'live-submit-queue',
+    notifyEmailAddress: process.env.NOTIFY_EMAIL_ADDRESS || 'live-notify-email-address'
+  },
+  serviceBusLocal: {
```

Comment:
Just for my knowledge why `serviceBusLocal`? Why do we need two? Tried searching in The PR description and could not find explanation.